### PR TITLE
add support for annotations

### PR DESCRIPTION
**Pull request by liambuchanan**

### CODE REVIEW

**Comment by ewgarci** [comment]
:+1:

File: kronjob/schema.json [hunk 1]
```diff
@@ -37,6 +37,10 @@
       "type": "object",
       "comment": "Abstract definition of a single job/cron job",
       "properties": {
+        "annotations": {
+          "type": "object",
+          "comment": "`annotations` in metadata for generated pod tempalte specs"
```
Comment (erikbern) [inline kronjob/schema.json]: *template

**Comment by imsky** [comment]
would be nice to have a basic test for labels + annotations

**Comment by liambuchanan** [comment]
@imsky yeah, I've got a list of things I'd like to add in the next few weeks. Tests for specific attributes in generated jobs is one of them.

**Comment by liambuchanan** [comment]
thanks @ewgarci

**Pull request merged**